docs/src/pages/_document.page.tsx

```diff
@@ -25,16 +25,20 @@ const getCSPContent = (context: Readonly<HtmlProps>) => {
       style-src 'self' 'unsafe-inline';
       font-src 'self' data:;
       frame-src *.codesandbox.io;
-      script-src 'unsafe-eval' 'self' '${cspInlineScriptHash}'
+      img-src 'self' cm.everesttech.net amazonwebservices.d2.sc.omtrdc.net;
+      connect-src 'self' *.shortbread.aws.dev dpm.demdex.net;
+      script-src 'unsafe-eval' 'self' '${cspInlineScriptHash}' a0.awsstatic.com;
```

Comment:
These are all what the tracking stuff needs. I figured these out by building, seeing an error, adding the domain to the proper place, repeat.